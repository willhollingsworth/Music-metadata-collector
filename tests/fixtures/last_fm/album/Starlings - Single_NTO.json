{
  "album": {
    "artist": "NTO",
    "mbid": "",
    "tags": {
      "tag": [
        {
          "url": "https://www.last.fm/tag/minimal",
          "name": "minimal"
        },
        {
          "url": "https://www.last.fm/tag/electronic",
          "name": "electronic"
        },
        {
          "url": "https://www.last.fm/tag/french",
          "name": "french"
        },
        {
          "url": "https://www.last.fm/tag/techno",
          "name": "techno"
        },
        {
          "url": "https://www.last.fm/tag/france",
          "name": "france"
        }
      ]
    },
    "name": "Starlings - Single",
    "image": [
      {
        "size": "small",
        "#text": ""
      },
      {
        "size": "medium",
        "#text": ""
      },
      {
        "size": "large",
        "#text": ""
      },
      {
        "size": "extralarge",
        "#text": ""
      },
      {
        "size": "mega",
        "#text": ""
      },
      {
        "size": "",
        "#text": ""
      }
    ],
    "tracks": {
      "track": [
        {
          "streamable": {
            "fulltrack": "0",
            "#text": "0"
          },
          "duration": 342,
          "url": "https://www.last.fm/music/NTO/Starlings+-+Single/Starlings",
          "name": "Starlings",
          "@attr": {
            "rank": 1
          },
          "artist": {
            "url": "https://www.last.fm/music/NTO",
            "name": "NTO",
            "mbid": "619f53fa-0f28-4f02-a313-9078979fd04c"
          }
        },
        {
          "streamable": {
            "fulltrack": "0",
            "#text": "0"
          },
          "duration": 488,
          "url": "https://www.last.fm/music/NTO/Starlings+-+Single/Starlings+(Henry+Saiz+Remix)",
          "name": "Starlings (Henry Saiz Remix)",
          "@attr": {
            "rank": 2
          },
          "artist": {
            "url": "https://www.last.fm/music/NTO",
            "name": "NTO",
            "mbid": "619f53fa-0f28-4f02-a313-9078979fd04c"
          }
        },
        {
          "streamable": {
            "fulltrack": "0",
            "#text": "0"
          },
          "duration": 365,
          "url": "https://www.last.fm/music/NTO/Starlings+-+Single/J%27oublie",
          "name": "J'oublie",
          "@attr": {
            "rank": 3
          },
          "artist": {
            "url": "https://www.last.fm/music/NTO",
            "name": "NTO",
            "mbid": "619f53fa-0f28-4f02-a313-9078979fd04c"
          }
        }
      ]
    },
    "listeners": "29",
    "playcount": "185",
    "url": "https://www.last.fm/music/NTO/Starlings+-+Single"
  }
}