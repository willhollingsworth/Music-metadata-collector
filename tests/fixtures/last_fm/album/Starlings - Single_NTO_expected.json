{
  "album_name": "Starlings - Single",
  "album_url": "https://www.last.fm/music/NTO/Starlings+-+Single",
  "listeners": "29",
  "playcount": "185",
  "artist_name": "NTO"
}